setup.py

```diff
@@ -1,29 +1,35 @@
 from setuptools import setup, find_packages
 
-
+LONG_DESCRIPTION = ''
 with open('README.md', 'r') as file:
-    long_description = '\n'.join(file.readlines()[3:])
-
+    LONG_DESCRIPTION = '\n'.join(file.readlines()[3:])
 
-requirements = [
+INSTALL_REQUIRES = (
     'pytz>=2018.4',
-    'requests>=2.19.1',
-    ]
+    'requests<3'
+)
+
+PROJECT = 'wapo-fec'
+AUTHOR = 'The Washington Post Newsroom Engineering Team'
```

Comment:
similarly here

Reply:
We would be the author of the PIP package. I think we should be explicit in the README that this is forked code

Reply:
Approve now given README changes